.github/workflows/nano_unit_tests_pytorch.yml

```diff
@@ -189,6 +190,7 @@ jobs:
             bash python/nano/dev/build_and_install.sh linux default false ${requirements[0]} -f https://software.intel.com/ipex-whl-stable
             pip install ${requirements[1]}
           fi
+          pip install numpy==1.23.4
```

Comment:
is `numpy>1.23.4` not supported by nano or any of its dependencies?

Reply:
when numpy>1.23.4, ` np.bool `has been completely abandoned. However, inc has `np.bool` itself, so if numpy>1.23.4, we can't even import inc.